content/documentation/edge/quick-start/index.adoc

```diff
@@ -46,9 +83,9 @@ Once you have the `istioctl` in your PATH, execute the following:
 istioctl install --set profile=demo
 ----
 
-=== Access to the UI
+==== Access to the UI
 
-The Istio client has an easy method to access to all of its UIs. To get access to the Kiali UI:
+The Istio `istioctl` client has an easy method to access to all of its UIs. To get access to the Kiali UI:
```

Comment:
Is this still true with the addons change?

Reply:
I think it is. All that does is create a port-forward. If you have kiali installed, however you installed it, that should work.